app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @pagy, @posts = pagy(@user.posts.includes(:comments), items: 2)
+    @pagy, @posts = pagy(@user.posts, items: 2)
   end
```

Comment:
- [ ] As requested by the previous reviewer, Please add `load_and_authorize_resource` top of your Post and Comment controller. By mistake, you added this to Post and Comment model, You should remove it from the models and add it to the controllers.